README.md

```diff
@@ -87,10 +92,15 @@ const migrations = require('ipfs-repo-migrations')
 const repoPath = 'some/repo/path'
 const currentRepoVersion = 7
 const latestVersion = migrations.getLatestMigrationVersion()
+const repoOptions = {
+  ... // the same storage backend/storage options passed to `ipfs-repo`
+}
```

Comment:
Is `repoOptions` still optional and we just need to make sure to pass it through if provided?

Reply:
It's been passed through by `ipfs-repo` [since it migrations were added](https://github.com/ipfs/js-ipfs-repo/blame/master/src/index.js#L335-L342).

Reply:
The readme api methods mention repoOptions as always optional, this is not the case as this change will throw errors. This doesn't matter for ipfs-repo since it is providing passing `repoOptions` to migrate and revert, but we should remove the optional jsdocs/readme as this **is** required.

Reply:
I've removed the optional notes in the readme.  Which is a bit odd because it makes the options object mandatory, but I think that's ok for now as we are the only consumer of this module and we pass an options object.  When I re-PR this against master I'll move the repo options out of the options object to make the options optional again.